package.json

```diff
@@ -60,10 +58,8 @@
 		"prestart": "npm run -s install-if-deps-outdated",
 		"start": "cross-env WC_ADMIN_PHASE=development npm run build:packages && cross-env WC_ADMIN_PHASE=development npm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"node ./bin/packages/watch.js\"",
 		"pretest": "npm run -s install-if-no-packages",
-		"test:e2e": "bash ./bin/wait-for-build.sh && ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js",
-		"test:e2e-dev": "bash ./bin/wait-for-build.sh && DEBUG=1 ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js --dev",
-		"test:e2e-debug": "bash ./bin/wait-for-build.sh && DEBUG=1 ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js --dev --debug",
 		"test": "./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json",
+		"test:e2e": "WP_VERSION=latest npm run build && npx wc-e2e docker:down && npx wc-e2e docker:up && npx wc-e2e test:e2e",
```

Comment:
Nice work on using `npx` instead, much better then the absolute path I added.
~~I do think we should remove the `npm run build` part though, that way we can run `npm run dev` on the side and run the tests whenever.
The docker teardown and build up doesn't seem to take too long (although not sure what it will be like when we are just running a specific spec file and debugging).~~

Saw your updated readme with just using the `npx wc-e2e` commands look great to me.